build.gradle.kts

```diff
@@ -82,8 +27,25 @@ nexusPublishing {
     }
 }
 
-tasks.maybeCreate("dokkaPublish").run {
+val dokkaPublish by tasks.registering {
     if (publicationChannels.any { it.isMavenRepository() }) {
         finalizedBy(tasks.named("closeAndReleaseSonatypeStagingRepository"))
     }
 }
+
+apiValidation {
+    ignoredProjects += setOf(
+        "search-component",
+        "compiler-dependency",
+        "kotlin-analysis",
+        "intellij-dependency",
+        "frontend",
+
+        "integration-tests",
+        "gradle",
+        "cli",
+        "maven",
+
+        "test-utils",
```

Comment:
> It's a shame it doesn't accept the project path. When I looked at `gradle`, I thought it was somehow referencing our gradle runner, not realizing there's a `gradle` project under integration tests

https://github.com/Kotlin/binary-compatibility-validator/issues/16

> also, we're not tracking the public API for the gradle plugin, but it's a good idea to start doing it. Can you run `apiDump` to generate it? it [fails the check now](https://github.com/Kotlin/dokka/actions/runs/4241592462/jobs/7372011661) because it's expecting the file, but can't find it

Yeah I was surprised that the Gradle Plugin was missing an API dump. And it's also odd that it hasn't been required until now. But anyway, I've added it now.

> Adding some comments would help

Done!